apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/OutpatientClaimTransformerTest.java

```diff
@@ -95,13 +95,11 @@ public void transformSampleARecord() throws FHIRException, IOException {
    * when run against the {@link StaticRifResource#SYNTHETIC_OUTPATIENT_1999_1999} {@link
    * OutpatientClaim}.
    *
-   * <p>Note: This test is normally disabled like other synthetic data tests
-   *
    * @throws FHIRException (indicates test failure)
    */
-  @Disabled
+  @Disabled("This test is normally disabled like other synthetic data tests")
```

Comment:
Do or will we ever run this? should these synthetic tests just be removed? (Are the synthetic records its trying to transform even valid compared to modern synthetic data?)

Reply:
Removed all